crates/proto/src/rr/rdata/opt.rs

```diff
@@ -826,16 +846,17 @@ mod tests {
         );
 
         let opt = read_rdata.unwrap();
-        let mut options = HashMap::default();
-        options.insert(
-            EdnsCode::Subnet,
-            EdnsOption::Subnet("0.0.0.0/0".parse().unwrap()),
-        );
-        options.insert(
-            EdnsCode::Cookie,
-            EdnsOption::Unknown(10, vec![0x0b, 0x64, 0xb4, 0xdc, 0xd7, 0xb0, 0xcc, 0x8f]),
-        );
-        options.insert(EdnsCode::Keepalive, EdnsOption::Unknown(11, vec![]));
+        let options = vec![
```

Comment:
Can you add a test case that covers this new use case? Otherwise this all looks good.

Reply:
I have added one using EDE as an example (currently used as unknown with 15 as its code).